examples/universal/z_get.cxx

```diff
@@ -86,7 +86,15 @@ int _main(int argc, char **argv) {
         done_signal.notify_all();
     };
 
+#if __cplusplus >= 201703L
```

Comment:
I thought we declare C++ 17 and higher, why do we need this check?

Reply:
(here and many places below)

Reply:
Under MSVC on windows the compiler refuses this aggregate initialization syntax 
```c++
session.get(keyexpr, "", on_reply, on_done, {.target = Z_QUERY_TARGET_ALL, .payload = Bytes::serialize(value)});
```
which is C++20 only
https://en.cppreference.com/w/cpp/language/aggregate_initialization

But it works in clang and gcc even in c++17 mode. I think it's good to keep this c++20 syntax in examples to demonstrate the best way to do the things, but we also need to provide fully compatible solution.

Reply:
Ok, got it!